examples/full-config/main.tf

```diff
@@ -1,7 +1,7 @@
 provider "cyral" {
   # client_id and client_secret may also be declared as env vars.
   # Please see provider docs for more info.
-  client_id = ""
+  client_id     = ""
   client_secret = ""
 
   control_plane = "mytenant.cyral.com:8000"
```

Comment:
Please remove the port declaration here.

Reply:
Removed this along with more changes in this commit: [49996e4](https://github.com/cyralinc/terraform-okta-idp/pull/17/commits/49996e4ba7d42a5cb10e1b2a10d5976ffcc243c5)